config/prow/build-cluster/README.md

```diff
@@ -1,7 +1,11 @@
+## Knative prow
+
+This directory contains prow configs hosted for Knative. This prow is bumped by
+[`bump.sh`](../bump.sh) script
```

Comment:
is it though?

Reply:
That script has been been deprecated for a long while. Currently they are using https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/knative/knative-autobump-config.yaml which is backed by https://github.com/kubernetes/test-infra/tree/master/prow/cmd/generic-autobumper